.github/workflows/ci.yml

```diff
@@ -221,21 +221,21 @@ jobs:
           kube_config=${{ github.workspace }}/deploy/helm-chart/kube-${{ github.run_id }}-helm
           make create-kind-cluster KIND_KUBE_CONFIG=${kube_config}
           echo "KUBECONFIG=${kube_config}" >> "$GITHUB_ENV"
-          kind load docker-image ${{ steps.ngf-meta.outputs.tags }} ${{ steps.nginx-meta.outputs.tags }}
+          kind load docker-image ghcr.io/nginxinc/nginx-gateway-fabric:${{ steps.ngf-meta.outputs.version }} ghcr.io/nginxinc/nginx-gateway-fabric/nginx:${{ steps.nginx-meta.outputs.version }}
```

Comment:
Where does the `version` variable get set in the outputs job above? I see `images` and `tags` as outputs, but not version (lines 184-186).

Also, do we need the `images` section anymore in the meta? Is it used anywhere?

Reply:
There may be misunderstanding on my part but from my understanding, in this file when it uses `with` it is taking in `images` and `tags` as inputs and docker/metadata-action linked [here](https://github.com/docker/metadata-action#inputs) allows for those inputs. In the docs, you can see the outputs below that you can just access after running docker/metadata-action, so you could also access `labels, json, and bake-file`.

You might still need the `images` section as it is used as the base name for the tags, and `outputs.tags` is still used when building the images.

Reply:
Ah I see, thanks for the context.